[2024-12-13T00:00:43.601+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-13T00:00:43.631+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecobalyse_dag.create_table manual__2024-12-13T00:00:36.199394+00:00 [queued]>
[2024-12-13T00:00:43.638+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecobalyse_dag.create_table manual__2024-12-13T00:00:36.199394+00:00 [queued]>
[2024-12-13T00:00:43.638+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2024-12-13T00:00:43.649+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_table> on 2024-12-13 00:00:36.199394+00:00
[2024-12-13T00:00:43.656+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=519) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-13T00:00:43.657+0000] {standard_task_runner.py:72} INFO - Started process 521 to run task
[2024-12-13T00:00:43.668+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecobalyse_dag', 'create_table', 'manual__2024-12-13T00:00:36.199394+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/data_transformations.py', '--cfg-path', '/tmp/tmpkbljairm']
[2024-12-13T00:00:43.669+0000] {standard_task_runner.py:105} INFO - Job 155: Subtask create_table
[2024-12-13T00:00:43.709+0000] {task_command.py:467} INFO - Running <TaskInstance: ecobalyse_dag.create_table manual__2024-12-13T00:00:36.199394+00:00 [running]> on host c8c82134bf48
[2024-12-13T00:00:43.959+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecobalyse_dag' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-13T00:00:36.199394+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-13T00:00:36.199394+00:00'
[2024-12-13T00:00:43.959+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-13T00:00:43.960+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-13T00:00:43.960+0000] {logging_mixin.py:190} INFO - Current task name:create_table state:running start_date:2024-12-13 00:00:43.631349+00:00
[2024-12-13T00:00:43.960+0000] {logging_mixin.py:190} INFO - Dag name:ecobalyse_dag and current dag run status:running
[2024-12-13T00:00:43.960+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-13T00:00:43.960+0000] {sql.py:278} INFO - Executing:  
                        DROP TABLE  IF EXISTS Ecobalyse_table;
                        CREATE TABLE IF NOT EXISTS Ecobalyse_table(
                           
                                acd NUMERIC,
                                cch NUMERIC,
                                etf NUMERIC, 
                                etf-c NUMERIC, 
                                fru NUMERIC, 
                                fwe NUMERIC,
                                htc NUMERIC,
                                htc-c NUMERIC, 
                                htn NUMERIC,
                                htn-c NUMERIC,
                                ior NUMERIC,  
                                ldu NUMERIC, 
                                mru NUMERIC, 
                                ozd NUMERIC,
                                pco NUMERIC, 
                                pma NUMERIC,
                                swe NUMERIC,
                                tre NUMERIC,
                                wtu NUMERIC,
                                ecs NUMERIC, 
                                pef NUMERIC, 
                                airTransportRatio NUMERIC, 
                                business TEXT,
                                countryDyeing TEXT,
                                countryFabric TEXT,
                                countryMaking TEXT, 
                                countrySpinning TEXT, 
                                fabricProcess TEXT,
                                makingComplexity' TEXT, 
                                makingDeadStock NUMERIC,
                                makingWaste NUMERIC, 
                                mass NUMERIC,
                                numberOfReferences NUMERIC,
                                physicalDurability NUMERIC,
                                price NUMERIC, 
                                product TEXT,
                                surfaceMass NUMERIC, 
                                traceability BOOLEAN,
                                upcycled BOOLEAN,
                                yarnSize NUMERIC
                           
                           
                           
                           
                           ); 
                    
[2024-12-13T00:00:43.970+0000] {base.py:84} INFO - Retrieving connection 'ecobalyse_database_connection'
[2024-12-13T00:00:43.995+0000] {base.py:84} INFO - Retrieving connection 'ecobalyse_database_connection'
[2024-12-13T00:00:44.000+0000] {sql.py:544} INFO - Running statement:  
                        DROP TABLE  IF EXISTS Ecobalyse_table;
                        CREATE TABLE IF NOT EXISTS Ecobalyse_table(
                           
                                acd NUMERIC,
                                cch NUMERIC,
                                etf NUMERIC, 
                                etf-c NUMERIC, 
                                fru NUMERIC, 
                                fwe NUMERIC,
                                htc NUMERIC,
                                htc-c NUMERIC, 
                                htn NUMERIC,
                                htn-c NUMERIC,
                                ior NUMERIC,  
                                ldu NUMERIC, 
                                mru NUMERIC, 
                                ozd NUMERIC,
                                pco NUMERIC, 
                                pma NUMERIC,
                                swe NUMERIC,
                                tre NUMERIC,
                                wtu NUMERIC,
                                ecs NUMERIC, 
                                pef NUMERIC, 
                                airTransportRatio NUMERIC, 
                                business TEXT,
                                countryDyeing TEXT,
                                countryFabric TEXT,
                                countryMaking TEXT, 
                                countrySpinning TEXT, 
                                fabricProcess TEXT,
                                makingComplexity' TEXT, 
                                makingDeadStock NUMERIC,
                                makingWaste NUMERIC, 
                                mass NUMERIC,
                                numberOfReferences NUMERIC,
                                physicalDurability NUMERIC,
                                price NUMERIC, 
                                product TEXT,
                                surfaceMass NUMERIC, 
                                traceability BOOLEAN,
                                upcycled BOOLEAN,
                                yarnSize NUMERIC
                           
                           
                           
                           
                           ); 
                    , parameters: None
[2024-12-13T00:00:44.001+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 284, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 489, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 549, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "-"
LINE 8:                                 etf-c NUMERIC, 
                                           ^

[2024-12-13T00:00:44.012+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-12-13T00:00:44.012+0000] {logging_mixin.py:190} INFO - Task start:2024-12-13 00:00:43.631349+00:00 end:2024-12-13 00:00:44.012354+00:00 duration:0.381005
[2024-12-13T00:00:44.013+0000] {logging_mixin.py:190} INFO - Task:<Task(PostgresOperator): create_table> dag:<DAG: ecobalyse_dag> dagrun:<DagRun ecobalyse_dag @ 2024-12-13 00:00:36.199394+00:00: manual__2024-12-13T00:00:36.199394+00:00, state:running, queued_at: 2024-12-13 00:00:36.211005+00:00. externally triggered: True>
[2024-12-13T00:00:44.013+0000] {logging_mixin.py:190} INFO - Failure caused by syntax error at or near "-"
LINE 8:                                 etf-c NUMERIC, 
                                           ^
[2024-12-13T00:00:44.013+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=ecobalyse_dag, task_id=create_table, run_id=manual__2024-12-13T00:00:36.199394+00:00, execution_date=20241213T000036, start_date=20241213T000043, end_date=20241213T000044
[2024-12-13T00:00:44.025+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-13T00:00:44.025+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 155 for task create_table (syntax error at or near "-"
LINE 8:                                 etf-c NUMERIC, 
                                           ^
; 521)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py", line 284, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 489, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 549, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "-"
LINE 8:                                 etf-c NUMERIC, 
                                           ^

[2024-12-13T00:00:44.075+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-12-13T00:00:44.105+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
